1st-gen/packages/picker/test/picker-responsive.test.ts

```diff
@@ -172,4 +175,130 @@ describe('Picker, responsive', () => {
             expect(popover, 'popover').to.not.be.null;
         });
     });
+
+    describe('touch device detection', () => {
+        afterEach(() => {
+            // Reset touch device simulation.
+            if (el && el.isTouchDevice) {
+                el.isTouchDevice.matches = false;
+            }
+        });
```

Comment:
This is incomplete. You have tests which also modify `el.isMobile.matches` in one test but don’t reset that value. Either restore the matchMedia stub in afterEach or explicitly reset both isTouchDevice and isMobile.